tools/reuse

```diff
@@ -5,14 +5,6 @@
 # SPDX-License-Identifier: BSD-2-Clause
 
 COPYRIGHT="Mewbot Developers <mewbot@quicksilver.london>"
-
 LICENSE="BSD-2-Clause"
 
-find . \
-	-path './.git' -prune -o \
-	-path './.reuse' -prune -o \
-	-path './LICENSES' -prune -o \
-	-path './venv' -prune -o \
-	-type f \
-	-exec \
-reuse addheader --merge-copyrights --copyright "$COPYRIGHT" --license "$LICENSE" --skip-unrecognised '{}' +
+reuse annotate --merge-copyrights --copyright "$COPYRIGHT" --license "$LICENSE" --skip-unrecognised --skip-existing --recursive .
```

Comment:
Query: Does this apply evenly across every file or just code files?

Reply:
Every file, but the tool is doing file type stuff to work out how to add a header.